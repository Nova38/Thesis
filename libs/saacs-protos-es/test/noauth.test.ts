import { assert, beforeAll, describe, expect, it } from 'vitest'

const res = Uint8Array.from([
  10, 214, 4, 10, 67, 10, 30, 115, 97, 97, 99, 115, 45, 99, 111, 108, 108, 101,
  99, 116, 105, 111, 110, 45, 49, 55, 49, 52, 56, 57, 48, 53, 49, 48, 54, 55,
  57, 18, 18, 115, 97, 97, 99, 115, 46, 97, 117, 116, 104, 46, 118, 48, 46, 82,
  111, 108, 101, 24, 2, 34, 11, 67, 111, 110, 116, 114, 105, 98, 117, 116, 111,
  114, 18, 142, 4, 10, 38, 116, 121, 112, 101, 46, 103, 111, 111, 103, 108, 101,
  97, 112, 105, 115, 46, 99, 111, 109, 47, 115, 97, 97, 99, 115, 46, 97, 117,
  116, 104, 46, 118, 48, 46, 82, 111, 108, 101, 18, 227, 3, 10, 30, 115, 97, 97,
  99, 115, 45, 99, 111, 108, 108, 101, 99, 116, 105, 111, 110, 45, 49, 55, 49,
  52, 56, 57, 48, 53, 49, 48, 54, 55, 57, 18, 11, 67, 111, 110, 116, 114, 105,
  98, 117, 116, 111, 114, 34, 179, 3, 10, 174, 3, 10, 26, 115, 97, 97, 99, 115,
  46, 98, 105, 111, 99, 104, 97, 105, 110, 46, 118, 48, 46, 83, 112, 101, 99,
  105, 109, 101, 110, 18, 143, 3, 24, 1, 34, 47, 10, 7, 112, 114, 105, 109, 97,
  114, 121, 18, 36, 10, 7, 112, 114, 105, 109, 97, 114, 121, 18, 7, 112, 114,
  105, 109, 97, 114, 121, 42, 16, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 21,
  20, 18, 24, 22, 23, 34, 53, 10, 9, 115, 101, 99, 111, 110, 100, 97, 114, 121,
  18, 40, 10, 9, 115, 101, 99, 111, 110, 100, 97, 114, 121, 18, 9, 115, 101, 99,
  111, 110, 100, 97, 114, 121, 42, 16, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19,
  21, 20, 18, 24, 22, 23, 34, 102, 10, 12, 103, 101, 111, 114, 101, 102, 101,
  114, 101, 110, 99, 101, 18, 86, 10, 12, 103, 101, 111, 114, 101, 102, 101,
  114, 101, 110, 99, 101, 18, 12, 103, 101, 111, 114, 101, 102, 101, 114, 101,
  110, 99, 101, 24, 1, 34, 48, 10, 7, 99, 111, 117, 110, 116, 114, 121, 18, 37,
  10, 7, 99, 111, 117, 110, 116, 114, 121, 18, 20, 103, 101, 111, 114, 101, 102,
  101, 114, 101, 110, 99, 101, 46, 99, 111, 117, 110, 116, 114, 121, 42, 4, 10,
  14, 18, 24, 42, 4, 10, 14, 18, 24, 34, 41, 10, 5, 116, 97, 120, 111, 110, 18,
  32, 10, 5, 116, 97, 120, 111, 110, 18, 5, 116, 97, 120, 111, 110, 42, 16, 10,
  11, 12, 13, 14, 15, 16, 17, 18, 19, 21, 20, 18, 24, 22, 23, 34, 41, 10, 5,
  108, 111, 97, 110, 115, 18, 32, 10, 5, 108, 111, 97, 110, 115, 18, 5, 108,
  111, 97, 110, 115, 42, 16, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 21, 20, 18,
  24, 22, 23, 34, 44, 10, 6, 103, 114, 97, 110, 116, 115, 18, 34, 10, 6, 103,
  114, 97, 110, 116, 115, 18, 6, 103, 114, 97, 110, 116, 115, 42, 16, 10, 11,
  12, 13, 14, 15, 16, 17, 18, 19, 21, 20, 18, 24, 22, 23, 34, 44, 10, 6, 105,
  109, 97, 103, 101, 115, 18, 34, 10, 6, 105, 109, 97, 103, 101, 115, 18, 6,
  105, 109, 97, 103, 101, 115, 42, 16, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19,
  21, 20, 18, 24, 22, 23, 42, 9, 10, 14, 18, 24, 14, 15, 11, 15, 17, 18, 0,
])

describe('Ho', () => {
  beforeAll(() => {})

  it('foo', () => {
    assert.equal(Math.sqrt(4), 2)
  })

  it('bar', () => {
    expect(1 + 1).eq(2)
  })

  it('snapshot', () => {
    expect({ foo: 'bar' }).toMatchSnapshot()
  })
})
