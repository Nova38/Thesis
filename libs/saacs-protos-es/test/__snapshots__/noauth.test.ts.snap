// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Ho > snapshot 1`] = `
{
  "foo": "bar",
}
`;

exports[`suite name > snapshot 1`] = `
{
  "foo": "bar",
}
`;
